README.md

```diff
@@ -1 +1,18 @@
-# logging-stack
+# logging-stack-configuration
+
+## Introduction
+
+These scripts are used to configure the logging stack
+
+## Roles
+
+- Kibana - Configure Kibana using the API
+
+### Kibana Role
+
+Makes requests against the Kibana API to manage an environments space and its index patterns.
+If an environment has not been defined within this role it will be removed from the logging-stack.
+
+## Running
+
+The playbooks are run within Concourse.
```

Comment:
I would change the wording here and specify that's it's run through Ansible or `ansible-runner` as a means of invoking it. It's not necessarily relevant that we're running it through Concourse

Reply:
Updated to say its invoked via the ansible-runner